src/app/util/helpers.js

```diff
@@ -228,3 +228,27 @@ export const isJarvis = () => {
 	}
 	return true;
 };
+
+/**
+ * Wrapper method to fetch customer's products
+ *
+ * @param {Object}   data         object of data
+ * @param {Function} passError    setter for the error in component
+ * @param {Function} thenCallback method to call in promise then
+ */
+
+export const productApiFetch = ( data, passError, thenCallback ) => {
```

Comment:
I'm not sure we need to extract `productApiFetch` method to the helpers just yet, since we're only using it once.